numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -10,13 +10,29 @@
 
 """
 from __future__ import division, absolute_import, print_function
+import textwrap
 
 docdict = {}
 
 def get(name):
     return docdict.get(name)
 
+# common parameter text to all ufuncs
+_params_text = textwrap.dedent("""
+    out : ndarray or tuple of ndarray, optional
+        Alternate array object(s) in which to put the result; if provided, it
+        must have a shape that the inputs broadcast to.
+    where : array_like, optional
+        Values of True indicate to calculate the ufunc at that position, values
+        of False indicate to leave the value in the output alone.
+
```

Comment:
To make it clearer that there are other arguments, I'd put this as part of the parameter list:
```
**kwargs
    For other keyword-only ...
```